### PR TITLE
[IAM-5212] add docs for setting up Okta SSO/SAML application

### PR DESCRIPTION
**Pull request by przemek-sl**

### CODE REVIEW

File: docs/basics/sso/configuring-sso-in-ms-azure-ad.md [hunk 1]
```diff
@@ -60,7 +60,7 @@ import useBaseUrl from '@docusaurus/useBaseUrl';
 
 16. Test the integration using the [IdP-initiated and the SP-initiated login](/basics/sso/logging-in-via-sso).
 
-    1. The IdP-initiated flow can be eaisly tested from the **Single sign-on** tab.
+    1. The IdP-initiated flow can be easily tested from the **Single sign-on** tab.
```
Comment (aliciakrebs) [inline docs/basics/sso/configuring-sso-in-ms-azure-ad.md:63]:
```suggestion
    1. The IdP-initiated flow can be tested from the **Single sign-on** tab.
```

File: docs/basics/sso/configuring-sso-in-okta.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+<p><span className="sauceDBlue">Enterprise Plans only</span></p>
+
+1. Log into **Okta** administrator panel, go to **Applications** and **Browse App Catalog**.
+   <img src={useBaseUrl('img/basics/sso/idp-config/okta/browse-app-catalog.png')} alt="Browse App Catalog" width="850" />
+
+2. Type **Sauce Labs** in the search box and choose the app **Sauce Labs**.
```
Comment (aliciakrebs) [inline docs/basics/sso/configuring-sso-in-okta.md:14]:
```suggestion
2. Enter  **Sauce Labs** in the search box and choose the app **Sauce Labs**.
```

File: docs/basics/sso/configuring-sso-in-okta.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+3. Click **Add Integration**.
+   <img src={useBaseUrl('img/basics/sso/idp-config/okta/add-integration.png')} alt="Add Integration" width="700" />
+
+4. Leave the default name or rename it and click **Next**.
+   <img src={useBaseUrl('img/basics/sso/idp-config/okta/general-settings.png')} alt="General Settings" width="800" />
+
+5. Click on the link **Identity Provider metadata**. The metadata will open in a new tab. Copy it and save as an xml file. Use this metadata file to set up the SSO integration at Sauce Labs in [Team Management Panel](/basics/sso/setting-up-sso#integrating-with-sauce-labs-service-provider).
```
Comment (aliciakrebs) [inline docs/basics/sso/configuring-sso-in-okta.md:22]:
```suggestion
5. Click the link **Identity Provider metadata**. The metadata will open in a new tab. Copy it and save as an XML file. Use this metadata file to set up the SSO integration at Sauce Labs in [Team Management Panel](/basics/sso/setting-up-sso#integrating-with-sauce-labs-service-provider).
```

File: docs/basics/sso/configuring-sso-in-okta.md [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+   <img src={useBaseUrl('img/basics/sso/idp-config/okta/download-metadata.png')} alt="Download Metadata" width="600" />
+
+6. In the **Credentials Details** section make sure that the value **Application username format** is a [valid email address](/basics/sso/setting-up-sso/#name-id). The default format for **Okta username** is email address, unless it was changed in Okta Admin Console.
+   <img src={useBaseUrl('img/basics/sso/idp-config/okta/app-username-format.png')} alt="Application Username Format" width="650" />
+
+7. Click **Done** in order to create the integration.
```
Comment (aliciakrebs) [inline docs/basics/sso/configuring-sso-in-okta.md:28]:
```suggestion
7. Click **Done** to create the integration.
```

File: docs/basics/sso/configuring-sso-in-okta.md [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+id: configuring-sso-in-okta
+title: Configuring SSO in Okta 
+sidebar_label: Configuring Okta
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+<p><span className="sauceDBlue">Enterprise Plans only</span></p>
+
+1. Log into **Okta** administrator panel, go to **Applications** and **Browse App Catalog**.
```
Comment (aliciakrebs) [inline docs/basics/sso/configuring-sso-in-okta.md:11]:
```suggestion
1. Log into **Okta** administrator panel, go to **Applications** and click **Browse App Catalog**.
```

**Review by aliciakrebs — APPROVED** [review]

File: docs/basics/sso/configuring-sso-in-ms-azure-ad.md (hunk 1 above)
Comment (przemek-sl) [reply docs/basics/sso/configuring-sso-in-ms-azure-ad.md:63]: fixed

File: docs/basics/sso/configuring-sso-in-okta.md (hunk 3 above)
Comment (przemek-sl) [reply docs/basics/sso/configuring-sso-in-okta.md:22]: fixed

File: docs/basics/sso/configuring-sso-in-okta.md (hunk 4 above)
Comment (przemek-sl) [reply docs/basics/sso/configuring-sso-in-okta.md:28]: fixed

File: docs/basics/sso/configuring-sso-in-okta.md (hunk 5 above)
Comment (przemek-sl) [reply docs/basics/sso/configuring-sso-in-okta.md:11]: fixed

**Pull request merged**